src/include/duckdb.h

```diff
@@ -2827,9 +2827,9 @@ enabled, nullptr is returned.
 
 * @param info A profiling information object
 * @param key The name of the metric setting to return the value for
-* @return The value of the metric setting. Must be freed with `duckdb_free`.
+* @return The value of the metric setting. Must be freed with `duckdb_destroy_value`.
```

Comment:
Should we add a comment about how to read the value? ie. `duckdb_get_varchar `. I'm pretty sure the comments here are sent to the docs

Reply:
Yes, fair remark, will push a comment. :)